deploy/base/clusterserviceversion.yaml

```diff
@@ -11,6 +11,9 @@ metadata:
     operatorframework.io/suggested-namespace: security-profiles-operator
     operators.openshift.io/valid-subscription: '["OpenShift Kubernetes Engine", "OpenShift Container Platform", "OpenShift Platform Plus"]'
     operatorframework.io/cluster-monitoring: "true"
+    operatorframework.io/arch.amd64: supported
+    operatorframework.io/arch.ppc64le: supported
```

Comment:
```suggestion
```

We don't need the arches. I think arm/amd64 are already supported and we are adding s390x.